src/top/dashboard.rs

```diff
@@ -73,11 +73,19 @@ impl HumanFormatter for i64 {
     }
 }
 
+static COMPONENT_HEADERS: [&str; 6] = [
+    "Name",
+    "Kind",
+    "Type",
+    "Events (Total/Sec)",
```

Comment:
I personally think the `(Total/Sec)` part of this is noise given I think the values in the column, like `36.66k (2.98k / s)` make it pretty clear.

Reply:
Agreed, I think it's pretty clear without.